File: solutions/0004-median-of-two-sorted-arrays/solution.py

class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        mergerd_array=nums1+nums2
        mergerd_array.sort()

        if len(mergerd_array)%2 != 0 :
            final= mergerd_array[len(mergerd_array)//2]
        elif len(mergerd_array) %2==0 :
            mid=len(mergerd_array)//2
            final=(mergerd_array[mid-1]+mergerd_array[mid])/2
        return final

File: solutions/0004-median-of-two-sorted-arrays/solution.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {
   let mergerd_array = [...nums1, ...nums2];
        mergerd_array.sort((a, b) => a - b);
    let final;
        if ((mergerd_array.length )%2 != 0){
             final= mergerd_array[Math.floor(mergerd_array.length/2)]
        }
        else {
           let  mid=mergerd_array.length / 2
         final=(mergerd_array[mid-1]+mergerd_array[mid])/2
        }
return final;
};
